### PR TITLE
FLAC support

### PR DESCRIPTION
**Pull request by KeitIG**
Finally :beer:

### CODE REVIEW

File: build.js [hunk 1]
```diff
@@ -38,14 +38,14 @@ function getDirectories(srcpath) {
...
-    'arch'           : ['ia32', 'x64'],
-    'electronVersion': '1.4.12',
-    'icon'           :  path.join('src', 'images', 'logos', 'museeks'),
-    'build-version'  :  app.version,
-    'app-version'    :  app.version,
+    'dir'           :  './',
+    'name'          :  'Museeks',
+    'platform'      : ['darwin', 'win32', 'linux'],
+    'arch'          : ['ia32', 'x64'],
+    'version'       :  '1.6.1',
```
Comment (YurySolovyov) [inline build.js]: latest beta is `1.6.2`

File: build.js [hunk 2]
```diff
@@ -38,14 +38,14 @@ function getDirectories(srcpath) {
 
 const options = {
     // required
-    'dir'            :  './',
-    'name'           :  'Museeks',
-    'platform'       : ['darwin', 'win32', 'linux'],
-    'arch'           : ['ia32', 'x64'],
-    'electronVersion': '1.4.12',
```
Comment (YurySolovyov) [inline build.js]: `version` option is deprecated and we should keep `electronVersion`

Comment (KeitIG) [inline build.js]: oh mb

**Comment by KeitIG** [comment]
1.6.1 is out of beta https://github.com/electron/electron/releases/tag/v1.6.1

**Comment by YurySolovyov** [comment]
Oh, now it is my bad, I thought they were keeping entire 1.6.x branch in beta

**Comment by KeitIG** [comment]
Nah, saw that last week :)

**Pull request merged**